### PR TITLE
Remove general tab from email pref form and align iems in form

### PR DESCRIPTION
**Pull request by karelhala**
Screenshot from 2020-03-20 17-15-03

### CODE REVIEW

**Comment by karelhala** [comment]
ping @mmenestr, I had to remove general tab, since that is not going to be used untill after summit.

**Comment by mmenestr** [comment]
Ok that's fine. Also I just tried looking at the alignment of "custom policies" and "Instant notifications" and "custom policies" is actually 2 pixels too low! So if the spacing above custom policies is 24px like we said in the call, could you actually switch it to 22px. sorry!

Screen Shot 2020-03-20 at 12 18 13 PM

File: src/data/general.json [hunk 1]
```diff
@@ -6,6 +6,15 @@
...
         "initializeOnMount": true,
         "description": "Be notified about known maintenance windows, new features and marketing annoucements for cloud.redhat.com"
+      }, {
+        "component": "text-field",
+        "label": "First name",
+        "name": "first-name"
+      },
+      {
+        "component": "text-field",
+        "label": "First name",
```
Comment (rvsia) [inline src/data/general.json]: Not sure what it does in the UI, but you have 2x First Name here. Maybe it should be Last name.

Comment (karelhala) [reply src/data/general.json]: This is just an example/testing file so we don't have to run API server all the time. But yeah, good point so others are not confused when using this repo.

**Pull request merged**